### ISSUE TITLE
Add support for Vue Router and Vuex

### ISSUE THREAD

**Issue by swznd** [fullcalendar/fullcalendar-vue#169]
Hi,

I think it's related to https://github.com/fullcalendar/fullcalendar-vue/issues/105

I cannot use this.$router is slot templates, it return undefined in vue3

**PostAction.vue:**
```
<template>
  <button @click="edit">Edit</button>
</template>

<script>
...
methods: {
  edit() {
    this.$router.push('/edit'); // return error cannot read property push of undefined
  }
}
...
</script>
```

**Calendar.vue**
```
<FullCalendar
  id="calendar"
  :options="calendarOptions"
>
  <template #eventContent="args">
    <PostAction />
  </template>
</FullCalendar>
```

it also happend in store and other plugins

I use fullcalendar v5.10.0

Runnable https://codesandbox.io/s/charming-neumann-p4ndk

**Comment by acerix** [comment]
I'm not sure this is a bug, are you looking to add support for Vue Router?

**Comment by swznd** [comment]
yes, support vue router and vuex will be nice

sometimes you need to add component that already have a logic with router or storage inside slot template

**Comment by abogaevski** [comment]
Hi,
also have tested this issue with Vue Composition Api.
useStore function return undefined and in the browser console `[Vue warn]: injection "store" not found.`
Also have noticed that custom components (e.g InlineSvg) also not working. Browser console log: `Failed to resolve component: inline-svg`
If it important i want to indicate that i have found a workaround. Just paste html  to v-slot directly. Error is showing if i use my component and send  props to this component

**Comment by arshaw** [comment]
Released in [v6.0.0](https://github.com/fullcalendar/fullcalendar/releases/tag/v6.0.0)
